### ISSUE TITLE
Copytext für Landingpage

### ISSUE THREAD

**Issue by d135-1r43** [hopps-app/hopps#69]
# User Story

Als Designer der Landingpage möchte ich einen Copytext zur Verfügung haben, um die Inhalte klar und ansprechend darzustellen.

# Akzeptanzkriterien

- Ein Absatz mit einer prägnanten Beschreibung des Projekts.
- Eine Tagline oder ein kurzer Satz als knappe Projektbeschreibung.
- Vorstellung der drei Hauptfeatures:
   1. Einfache Buchhaltung
   2. Flexibles Struktursystem "Bommel"
   3. Mobile App für alle Plattformen
- Der erstellte Text ist als Kommentar in diesem Ticket dokumentiert.

# Abhängigkeiten

Wird benötigt für #67 und #69, verblockt aber nicht vollständig.

**Comment by sk2675** [comment]
@lukas-leonhardt ich haben einen ersten Entwurf, würden gerne dann nochmal mit dir das einmal durchgehen.

**Comment by d135-1r43** [comment]
Ist das fertig? Wenn ja, wo finde ich den Text?

**Comment by sk2675** [comment]
Hier der erste richtige Entwurf 
https://docs.google.com/document/d/1uKKAoHrEgPgknRFYjljEW2hydMf0ta8njcv0eKd9QYI/edit?usp=sharing

**Comment by manuelhummler** [comment]
Später sollten wir hier auf jeden Fall noch hervorheben, dass man das ganze einfach selbst hosten kann, oder cloud basiert buchen kann. ähnlich wie bei [kimai](https://www.kimai.org/de/) und [openproject ](https://www.openproject.org/de/preise/).

**Comment by d135-1r43** [comment]
Ich habe ein wenig kommentiert. Erstens gern auf Microsoft umziehen, Google nutzen wir nicht mehr. Zweitens bitte mehr Struktur schaffen durch einheitliches Format und klare Hierarchien.

**Comment by sk2675** [comment]
Copytexts can be found at /Open Project e.V\Hopps - General\06_Dokumentation

**Comment by eliashehme** [comment]
@d135-1r43 Wie sieht es denn hier aus. Kann diese To-Do geschlossen werden?

**Comment by d135-1r43** [comment]
Nope, die Texte gefallen mir noch nicht. Ich hab grad einige Kommentare ergänzt.

**Comment by sk2675** [comment]
Texte wurden verbessert. Nochmal checken biite! @d135-1r43

**Comment by d135-1r43** [comment]
Danke. Ich hab nochmal kommentiert.

**Comment by d135-1r43** [comment]
Ist das Feedback inzwischen umgesetzt?

**Comment by d135-1r43** [comment]
Auf Done schiebt nur der PO.

**Comment by sk2675** [comment]
Wollte das Ticket damit löschen, da wir ja die Tickets jetzt extern im Management-Team tracken.

**Comment by d135-1r43** [comment]
Okay, dann passt es.
